src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -399,10 +399,14 @@ REPORT_CACHE = 'redis'
 
 redis_cache = {
     'BACKEND': 'django_redis.cache.RedisCache',
+{% set redis_url = 'redis://:' ~ localsettings.REDIS_HOST ~ ':' ~ localsettings.REDIS_PORT %}
+{% if localsettings.get('REDIS_PASSWORD') %}
+{% set redis_url = 'redis://:' ~ localsettings.REDIS_PASSWORD ~ '@' ~ localsettings.REDIS_HOST ~ ':' ~ localsettings.REDIS_PORT %}
```

Comment:
This still makes a diff that I think would cause a problem; it inserts an `:` that shouldn't be there:
```diff
-    'LOCATION': 'redis://10.201.40.219:6379/0',
+    'LOCATION': 'redis://:10.201.40.219:6379/0',
```